daemon/graphdriver/vfs/driver.go

```diff
@@ -69,11 +81,32 @@ func (d *Driver) Cleanup() error {
 	return nil
 }
 
+func (d *Driver) parseOptions(options []string) error {
+	for _, option := range options {
+		key, val, err := parsers.ParseKeyValueOpt(option)
+		if err != nil {
+			return errdefs.InvalidParameter(err)
+		}
+		switch key {
+		case "size":
+			size, err := units.RAMInBytes(val)
+			if err != nil {
+				return errdefs.InvalidParameter(err)
+			}
+			if err = d.setQuotaOpt(uint64(size)); err != nil {
+				return errdefs.InvalidParameter(errors.Errorf("failed to set option size for vfs: %v", err))
```

Comment:
This should wrap the original error;

```suggestion
				return errdefs.InvalidParameter(errors.Wrap(err, "failed to set option size for vfs"))
```

Reply:
yes, and I have updated.